lisp/mastodon-tl.el

```diff
@@ -146,6 +155,71 @@ Return value from boosted content if available."
   (or (cdr (assoc field (cdr (assoc 'reblog toot))))
       (cdr (assoc field toot))))
 
+(defun mastodon-tl--relative-time-details (timestamp &optional current-time)
+  "Returns cons of (descriptive string . next change) for the TIMESTAMP.
```

Comment:
Agreed.  Let's keep it separate.  It's a habbit I picked up at work.  3rd person is the correct Java style here.